CHANGELOG.md

```diff
@@ -2,11 +2,21 @@
 
 ## UNRELEASED
 
-## v0.0.9 - 2023-01-23
+### Fixed
+
+- Do not unregister boilerplates store module beforeDestroy
```

Comment:
In order to keep things consistent, I'd suggest to link the PR and the author here as in the entries below

Reply:
done in 75c08c79442446578ed2b77d0236c270a9f6a533